presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -269,4 +271,16 @@ public FeaturesConfig setRegexLibrary(RegexLibrary regexLibrary)
         this.regexLibrary = regexLibrary;
         return this;
     }
+
+    public boolean isCharPadSpaces()
+    {
+        return charPadSpaces;
+    }
+
+    @Config("char.pad-spaces")
```

Comment:
Added description.

`PAD SPACES` is part of collation. We can assume just one collation without `PAD SPACES`, however it is useful to have this option selectable especially when Presto is being queried from other database (e.g. Teradata). Different databases do strings comparison differently, therefore with this option we can adjust for their behavior.